src/index.html

```diff
@@ -11,7 +11,170 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="POST"
+      onsubmit="onSubmit()"
+    >
+      <fieldset>
+        <legend>Personal information</legend>
+        <div class="formElement">
+          <label for="surname">Surname:</label>
+          <input
+            type="text"
+            name="surname"
+            id="surname"
+          >
+        </div>
+        <div class="formElement">
+          <label for="name">Name:</label>
+          <input
+            type="text"
+            name="name"
+            id="name"
+          >
+        </div>
+        <div class="formElement">
+          <label for="age">How old are You?</label>
+          <input
+            type="number"
+            name="age"
+            id="age"
+            value="12"
+            min="12"
+            max="100"
+          >
+        </div>
+        <div class="formElement">
+          <label for="dateOfBirth">Full date of birth:</label>
+          <input
+            type="date"
+            name="dateOfBirth"
+            id="dateOfBirth"
+          >
+        </div>
+        <div class="formElement">
+          <label for="terms">I accept the term of the agreement</label>
+          <input
+            type="checkbox"
+            name="terms"
+            id="terms"
+            required
+          >
+        </div>
+      </fieldset>
+      <fieldset>
+        <legend>Registration</legend>
+        <div class="formElement">
+          <label for="email">E-mail:</label>
+          <input
+            type="email"
+            name="email"
+            id="email"
+            placeholder="email@example.com"
+            required
+          >
+        </div>
+        <div class="formElement">
+          <label for="pass">Password:</label>
+          <input
+            type="password"
+            name="pass"
+            id="pass"
+            required minlength="5"
+            maxlength="16"
+          >
+        </div>
+      </fieldset>
+      <fieldset>
+        <legend>An interesting fact about you!</legend>
+        <div class="formElement">
+          Do you love cats?
+          <label for="cats1">
+            <input
+              type="radio"
+              name="cats"
+              id="cats1"
+              value="yes"
+            >
+            Yes
+          </label>
+          <label for="cats2">
+            <input
+              type="radio"
+              name="cats"
+              id="cats2"
```

Comment:
Using numbers for yes/no buttons is a bit weird, let's make it more specific
```suggestion
              id="catsNo"
```